torch/csrc/jit/passes/quantization.h

```diff
@@ -96,5 +96,10 @@ TORCH_API void QuantFusion(std::shared_ptr<Graph>& graph);
  */
 TORCH_API void FoldConvBatchNorm2d(const script::Module& module);
 
+/** \brief Fold quantize function call into module
```

Comment:
This comment doesn't really help: please explain what exactly we're folding, for what modules and in what calls. The name also might be more specific I think.